tree/ntuple/v7/src/RField.cxx

```diff
@@ -160,6 +160,8 @@ std::string GetNormalizedType(const std::string &typeName) {
    if (normalizedType.substr(0, 8) == "variant<") normalizedType = "std::" + normalizedType;
    if (normalizedType.substr(0, 5) == "pair<") normalizedType = "std::" + normalizedType;
    if (normalizedType.substr(0, 6) == "tuple<") normalizedType = "std::" + normalizedType;
+   if (normalizedType.substr(0, 7) == "bitset<")
+      normalizedType = "std::" + normalizedType;
```

Comment:
It's `clang-format` choice, but we should be probably consistent, at least for lines in the same block.